src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/PackageAnnotationCheck.java

```diff
@@ -86,7 +86,7 @@ public void visitToken(final DetailAST ast) {
             getFileContents().inPackageInfo();
 
         if (containsAnnotation && !inPackageInfo) {
-            log(ast.getLine(), MSG_KEY);
+            log(ast.getLineNo(), MSG_KEY);
```

Comment:
This actually caused a change in the tests. We populate LineNo but ANTLR populates Line.
In the test, ANTLR didn't populate this line number, maybe we created the AST ourselves. We did populate LineNo and so the test now shows the correct line number.